README.md

```diff
@@ -1,16 +1,15 @@
-# charmed-kafka-snap
+# Charmed Kafka snap repository.
 
-Charmed Kafka snap repository.
-
-This snap is based on the kafka snap: https://git.launchpad.net/kafka-snap/
+[![CharmHub Badge](https://snapcraft.io/charmed-kafka/badge.svg)](https://snapcraft.io/charmed-kafka)
+[![Release](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-kafka-snap/actions/workflows/publish.yaml)
```

Comment:
We have `build & test` CI. I think that could be added here.

Reply:
I believe the problem here is that we don't have the "build & test" in the release pipeline (although maybe we should) but only in the PRs. So the workflows only include feature branches, see [here](https://github.com/canonical/charmed-kafka-snap/actions/workflows/ci.yaml)